packages/fxa-settings/package.json

```diff
@@ -257,7 +257,9 @@
     "webpack": "^5.84.1"
   },
   "nx": {
-    "implicitDependencies": ["crypto-relier"],
+    "implicitDependencies": [
```

Comment:
Any reason to include this change here instead of the next time we touch this file or did this happen on commit hook?

Reply:
I did not do this intentionally, I'll remove since it's unrelated.